<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>DEVELOPERS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="font">
               <font>
                <family>MS Sans Serif</family>
                <pointsize>8</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>search name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/newPrefix/search.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>view</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>1.Lawrence Reneck</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>3. Davis Banda</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>2. Oscar Nguluwe</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>view</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>view</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QPushButton" name="pushButton_5">
                 <property name="text">
                  <string>view</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>view</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>5. Biata Chirwa</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>4. Davis Banda</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QWidget" name="widget_5" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>NAME</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>DETAILS</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QTextEdit" name="textEdit">
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Add a developer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pictures/yo.qrc"/>
 </resources>
 <connections/>
</ui>
